web/api/v1/pod.ts

```diff
@@ -1,26 +1,69 @@
 import { V1Pod, V1PodList } from "@kubernetes/client-node";
-import { instance } from "@/api/v1/index";
+import { k8sInstance, namespaceURL } from "@/api/v1/index";
+import axios from "axios";
 
 export const applyPod = async (req: V1Pod, onError: (_: string) => void) => {
   try {
-    const res = await instance.post<V1Pod>(`/pods`, req);
+    if (!req.metadata?.name) {
+      onError("metadata.name is not provided");
+      return;
+    }
+    const res = await k8sInstance.patch<V1Pod>(
+      namespaceURL + `/pods/${req.metadata.name}?fieldManager=simulator`,
+      req,
+      { headers: { "Content-Type": "application/strategic-merge-patch+json" } }
+    );
     return res.data;
   } catch (e: any) {
-    onError(e);
+    if (axios.isAxiosError(e) && e.response && e.response.status === 404) {
```

Comment:
I think this 404 error should be resolved by using server side apply. You currently use patch.
So, use `Content-Type` to `application/apply-patch+yaml` instead.
https://kubernetes.io/docs/reference/using-api/server-side-apply/#api-endpoint

Reply:
And if the error is gone, you can delete `createPod` :)

Reply:
Thank you, this opportunity gave me learn and understand server side apply!